libtransmission/announcer-http.cc

```diff
@@ -118,7 +118,7 @@ bool handleAnnounceResponse(tr_web::FetchResponse const& web_response, tr_announ
     if (status != HTTP_OK)
     {
         auto const* const response_str = tr_webGetResponseStr(status);
-        response->errmsg = fmt::format(FMT_STRING("Tracker HTTP response {:d} ({:s}"), status, response_str);
+        response->errmsg = fmt::format(FMT_STRING("Tracker HTTP response {:d} ({:s})"), status, response_str);
```

Comment:
Oog, I was really uncomfortable having to choose between "fix the parenthesis" and "preserve the string freeze" -- but looks like the corrected version is also on line 529 so ... we should be OK?

Reply:
I didn't see any existing translation for either string. Maybe I didn't look at the right place?